### PR TITLE
5007/fhir cleanup

### PR DESCRIPTION
**Pull request by DanielSass**
# BACKEND PULL REQUEST

## Related Issue

- #5007 (feedback from sync with Report Stream engineering) 

## Changes Proposed

- Misc updates and fixes to our fhir mapping

## Additional Information

- Unless I'm missing something we don't have a `country` value for Orgs or Providers, defaulting to "USA"
- Still figuring out how exactly to set the Observation.method with device information.

## Testing

- Existing unit tests + verify that new fields / code have new tests

### CODE REVIEW

**Review by BobanL — APPROVED** [review]

File: backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java [hunk 1]
```diff
@@ -144,14 +145,16 @@ public static ContactPoint convertToContactPoint(ContactPointUse contactPointUse
     return convertToContactPoint(contactPointUse, ContactPointSystem.PHONE, number);
   }
 
-  public static List<ContactPoint> convertEmailsToContactPoint(List<String> emails) {
+  public static List<ContactPoint> convertEmailsToContactPoint(
+      ContactPointUse use, List<String> emails) {
     return emails.stream()
-        .map(FhirConverter::convertEmailToContactPoint)
+        .map(s -> convertEmailToContactPoint(use, s))
```
Comment (emmastephenson) [inline backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java:151]: nit: s is not a very descriptive variable name

**Review by emmastephenson — APPROVED** [review]
Looks good! Can confirm that as far as I know, we don't track country for providers and just assume they're all US-based.

**Review by emmastephenson — APPROVED** [review]
LGTM, thanks!

**Review by BobanL — APPROVED** [review]
👍

**Pull request merged**